cmd/podman/create.go

```diff
@@ -940,3 +964,52 @@ func createContainerFromCreateConfig(r *libpod.Runtime, createConfig *cc.CreateC
 	}
 	return ctr, nil
 }
+
+func makeHealthCheckFromCli(c *cliconfig.PodmanCommand) (*manifest.Schema2HealthConfig, error) {
+	inCommand := c.String("healthcheck-command")
+	inInterval := c.String("healthcheck-interval")
+	inRetries := c.Uint("healthcheck-retries")
+	inTimeout := c.String("healthcheck-timeout")
+	inStartPeriod := c.String("healthcheck-start-period")
+
+	// Every healthcheck requires a command
+	if !c.Flags().Changed("healthcheck-command") {
+		return nil, errors.New("Must define a healthcheck command for all healthchecks")
+	}
+
+	cmd, err := shlex.Split(inCommand)
+	if err != nil {
+		return nil, err
+	}
+	hc := manifest.Schema2HealthConfig{
```

Comment:
Shouldn't we be loading this with the defaults to begin with?

Reply:
can we please defer this stuff to phase3 when those values are actually used? they are not being used right now.